.github/workflows/specs.yml

```diff
@@ -22,18 +48,42 @@ jobs:
       - run: source ~/.rvm/scripts/rvm && gem install tf -v '>=0.4.1'
 
       - name: named_ruby_and_gemsets_comment_test
-        if: ${{ !contains(matrix.os, 'macos') }} # works on local, but fails in CI, needs to be investigated
         run: source ~/.rvm/scripts/rvm && tf --text tests/long/named_ruby_and_gemsets_comment_test.sh
 
+  tests_long__ruby_prepare_mount_comment:
+    name: Ruby prepare mount comment test
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    env:
+      TERM: ansi
+      RVM_SKIP_BREW_UPDATE: true
+    steps:
+      - uses: actions/checkout@v4
+      - run: ./install
+      - run: source ~/.rvm/scripts/rvm && rvm use 2.7.7 --install --default
+      - run: source ~/.rvm/scripts/rvm && gem install tf -v '>=0.4.1'
+
       - name: ruby_prepare_mount_comment_test
-        if: ${{ !contains(matrix.os, 'macos') }} # https://github.com/rvm/rvm/issues/4937
         run: source ~/.rvm/scripts/rvm && tf --text tests/long/ruby_prepare_mount_comment_test.sh
 
+  tests_long__truffleruby_comment:
+    name: TruffleRuby comment test
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+    runs-on: ${{ matrix.os }}
+    env:
+      TERM: ansi
+      RVM_SKIP_BREW_UPDATE: true
+    steps:
+      - uses: actions/checkout@v4
+      - run: ./install
+      - run: source ~/.rvm/scripts/rvm && rvm use 2.7.7 --install --default
```

Comment:
Isnt this one for trufflefurby?

Reply:
I don't want to guess, so let's check it out.

Reply:
The [tests didn't pass](https://github.com/rvm/rvm/actions/runs/8940912966), so no.

Reply:
I think it's more the opposite: `truffleruby_comment_test.sh` might not need `2.7.7` to be installed. But it feels something out of scope and might not hold either, so I wouldn't change that in this PR.